build.gradle

```diff
@@ -66,6 +68,9 @@ dependencies {
 		transitive = false
 	}
 	compile 'net.sf.jopt-simple:jopt-simple:5.0.3'
+
+	compile 'org.apache.logging.log4j:log4j-core:2.8.1'
```

Comment:
This is the log4j version used by minecraft 19w40a, ideally we should remove this dep from loader all togeather.